### PR TITLE
Link latest Kubernetes blog post

### PR DESCRIPTION
**Pull request by saschagrunert**
cc @kubernetes/sig-release @sftim

Preview: https://deploy-preview-525--kubernetes-contributor.netlify.app/resources/

### CODE REVIEW

File: content/en/resources/website/.gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# latest release blog
+!.gitignore
+*
```
Comment (sftim) [inline content/en/resources/website/.gitignore:3]: I don't understand this change. What's happening here?

**Comment by sftim** [comment]
I don't think this PR helps @saschagrunert; it won't boost the SEO for https://kubernetes.io/blog/2024/08/13/kubernetes-v1-31-release/

**Comment by saschagrunert** [comment]
Let me try to add it as external link.

File: content/en/resources/website/.gitignore (hunk 1 above)
Comment (saschagrunert) [reply content/en/resources/website/.gitignore]: This is not required any more.

File: content/en/resources/_index.md [hunk 2]
```diff
@@ -1,8 +1,10 @@
...
-title:  "Resources"
+title: "Resources"
 description: "Kubernetes Community Resources"
 type: docs
 menu:
   main:
     weight: 50
 ---
+
+##### [Kubernetes v1.31: Elli](https://kubernetes.io/blog/2024/08/13/kubernetes-v1-31-release)
```
Comment (saschagrunert) [inline content/en/resources/_index.md:10]: Hm, I have no idea how to format that link properly into the section index :thinking:

**Comment by saschagrunert** [comment]
@kubernetes/sig-release-leads PTAL

**Comment by jberkus** [comment]
I'm trying to figure out how the entry for 1.32 Timeline got into this page, when it's not in your PR, but it's also not in Main.

If we're going to have 1.31 and 1.32 information, the two items should be next to each other.

**Comment by saschagrunert** [comment]
> If we're going to have 1.31 and 1.32 information, the two items should be next to each other.

Yep, now they're next to each other.

File: content/en/resources/release-1.31.md
Comment (sftim) [inline content/en/resources/release-1.31.md:1]: Now that https://www.k8s.dev/resources/release/ mentions v1.32, we can simply close this PR.

Comment (jberkus) [reply content/en/resources/release-1.31.md:1]: Wait, do you mean 1.31?

Comment (sftim) [reply content/en/resources/release-1.31.md:1]: I mean 1.32; see https://www.k8s.dev/resources/release/

**Pull request closed without merging**